lix/packages/sdk/src/query-utilities/get-lowest-common-ancestor.ts

```diff
@@ -12,8 +12,16 @@ export async function getLowestCommonAncestor(args: {
 	sourceLix: LixReadonly;
 	targetLix: LixReadonly;
 }): Promise<Change | undefined> {
-	// the change has no parent (it is the root change)
-	if (!args.sourceChange?.parent_id) {
+	const parentEdges = await args.sourceLix.db
+		.selectFrom("change_edge")
+		.selectAll()
+		.where("child_id", "=", args.sourceChange.id)
+		// TODO https://github.com/opral/lix-sdk/issues/105
+		// .where(isInSimulatedCurrentBranch)
```

Comment:
One of the TODOS that should be tackled after the branch milestone. 

- branching will influence the logic
- working on this now is a waste of time